lib/shopify_api.rb

```diff
@@ -22,3 +21,11 @@ module ShopifyAPI
 require 'shopify_api/countable'
 require 'shopify_api/resources'
 require 'shopify_api/session'
+require 'shopify_api/connection'
+
+# ActiveResource::Base.connection_class was introduced in https://github.com/rails/activeresource/pull/222
```

Comment:
@mikeyhew can you remove this comment? It's useful information for the pull request, but doesn't seem necessary in the codebase.